File: 2-13认证杯/08-20年挑战赛优秀论文汇总/2013第六届一阶段优秀论文/第一阶段/b/特等奖/1023队/code/framework/matlab_skeleton/TrainModel.m
function TrainModel(classlist)

% first get the classes from file
[ classes_names, nb_elem_in_class, classes ] = ReadClassData(classlist);

% do all the processing here...


% write results
modelname = [ classlist, '.model' ];
f = fopen(modelname, 'w');

for i = 1:length(classes_names)
	% write a model
	%......
end

fclose(f);

File: 2-13认证杯/08-20年挑战赛优秀论文汇总/2013第六届一阶段优秀论文/第一阶段/b/特等奖/1023队/code/framework/python_skeleton/TrainModel.py
#!/usr/bin/python

import ClassData
import sys

if len(sys.argv) != 3:
    print 'usage: ', sys.argv(0), ' classfile type'
    print 'where classfile is the path to the file containing the classes definitions'
    print 'and type is the type of contest (one of \'artist\', \'genre\' or \'similarity\')'
    sys.exit(1)

# get arguments from commandline
classfile = sys.argv(1)
contesttype = sys.argv(2)

# first get the classes from file
classes = read_class_data(classlist)

# train the model here...






# write the results
modelfile = classfile + '.model.' + contesttype
out = open(modelfile, 'w')
for i in range(len(classes)):
    # write a model here...
